.github/workflows/sanity.yml

```diff
@@ -11,16 +11,14 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         version: [8, 11, 17]
-        impl: [hotspot]
+        distribution: [temurin]
```

Comment:
`distribution` is required by setup-java action. The parameter name can keep as before.